### ISSUE TITLE
[@types/react-redux] Version 7.0.2

### ISSUE THREAD

**Issue by Pong420** [DefinitelyTyped/DefinitelyTyped#33904]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/xxxx` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @....

@jamesreggio @JounQin 

After updating from `@types/react-redux 7.0.1` to `@types/react-redux 7.0.2 `. The type of `Connected` is `any` 
```js
const Connected = connect()(() => <div />);
```
Is it a bug? Or i need to define myself
```js
const Connected: React.FC = connect()(() => <div />);
```

**Comment by jamesreggio** [comment]
There's a bug in 7.0.2 that is likely to blame.

The fix is landing in #33919, and the main issue is #33690.

**Comment by Pong420** [comment]
@jamesreggio seems 7.0.5 not fix the issue
